### PR TITLE
Update errors.go

### PR DESCRIPTION
**Pull request by maltsev25**
**fix for aliases**
translate work incorrect for registered aliases

## Fixes Or Enhances

**Make sure that you've checked the boxes below before you submit PR:**
- [x] Tests exist or have been written that cover this particular change.

@go-playground/validator-maintainers

### CODE REVIEW

**Comment by maltsev25** [comment]
@deankarn Can you please review this PR?

File: errors.go [hunk 1]
```diff
@@ -260,19 +260,22 @@ func (fe *fieldError) Error() string {
...
 	}
 
-	fn, ok := m[fe.tag]
+	fn, ok = m[fe.tag]
 	if !ok {
 		fn, ok = m[fe.actualTag]
 		if !ok {
 			return fe.Error()
 		}
-		return fe.Error()
```
Comment (hjwk) [inline errors.go:271]: Looks like this is the only line that needs to change ?

**Review by hjwk — CHANGES_REQUESTED** [review]
Can you squash your three commits into 1 ?

File: errors.go (hunk 1 above)
Comment (maltsev25) [reply errors.go:271]: You need to check variable `m` with 2 different tags `fe.tag` and `fe.actualTag`

Comment (hjwk) [reply errors.go:271]: Ah, correct, got confused when looking at the diffs, with 1 commit it is clearer now

**Review by hjwk — APPROVED** [review]

**Comment by maltsev25** [comment]
@deankarn Can you please review this PR?

**Comment by maltsev25** [comment]
@deankarn Can you please review this PR?

**Review by deankarn — APPROVED** [review]
LGTM

I will try to merge and release this weekend, have a lot going on so may be a few days.

**Pull request merged**